cometd-java/cometd-java-server/cometd-java-server-common/src/main/java/org/cometd/server/http/JSONHttpTransport.java

```diff
@@ -90,19 +91,21 @@ private void process(TransportContext context, List<ServerMessage.Mutable> messa
         }
     }
 
-    private abstract class AbstractReader {
+    private abstract class AbstractReader extends Invocable.Task.Abstract {
```

Comment:
This class is in the `cometd-java-server-common` module but `Invocable` is a Jetty-specific class. Shouldn't the change be done in `JettyCometDRequest` instead?

Reply:
As discussed: this is a minor issue since CometD already depends on `jetty-util` where `Invocable` lies, and moving the `NON_BLOCKING` `InvocationType` change to the jetty-specific module would either require an allocation or some substantial effort while this implementation does the job with little effort and no adverse side-effect.

So we're keeping this change as-is.